.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - "6"
-  - "4"
-  - "node"
+  - "8"
```

Comment:
Are we not going to use `lts/*` here?

Reply:
Trevor parses the .travis.yml file differently than Travis CI does. It only accepts "stable", "node", or "X.Y.Z" (not "lts", etc). "stable" and "node" are equivalent, and they are determined by downloading and parsing nodejs.org/dist, to find the latest version with an even minor number. Also, Trevor ignores our minor and patch version specifications, so "X.Y.Z" does the equivalent of "nvm use X".